lib/pharos/resources/calico/20-configmap.yml.erb

```diff
@@ -7,14 +7,11 @@ metadata:
 data:
   ipv4_pool_cidr: "<%= ipv4_pool_cidr %>"
 
-
-
   # Configure the Calico backend to use.
   calico_backend: "bird"
 
-  # TODO Do we need mtu setting?
   # Configure the MTU to use
-  veth_mtu: "1440"
+  veth_mtu: "1500"
```

Comment:
Not sure if this is the correct number for all modes, we use ipip by default which most likely should use `1440` ..